sources/packages/api/src/route-controllers/institution-locations/models/application.dto.ts

```diff
@@ -30,3 +40,43 @@ export class ActiveApplicationSummaryAPIOutDTO {
   applicationStatus: ApplicationStatus;
   fullName: string;
 }
+
+/**
+ * Transformation util for Active application.
+ * @param application application object.
+ * @param offering offering object.
+ * @returns active application data dto
+ */
+export const transformToActiveApplicationDataAPIOutDTO = (
+  application: Application,
+  offering: EducationProgramOffering,
+): ActiveApplicationDataAPIOutDTO => {
+  return {
+    applicationStatus: application.applicationStatus,
+    applicationNumber: application.applicationNumber,
+    applicationOfferingIntensity: offering.offeringIntensity,
+    applicationOfferingStartDate: dateString(offering.studyStartDate),
```

Comment:
Not really part of this PR but we need to change the date format used by `dateString` to the same from Vue that we recently updated to be exactly what is requested by Figma.